src/rust/engine/testutil/src/lib.rs

```diff
@@ -60,6 +60,10 @@ pub fn make_file(path: &Path, contents: &[u8], mode: u32) {
   file.set_permissions(permissions).unwrap();
 }
 
+pub fn make_link(path: &Path, target: &Path) {
```

Comment:
This is only used in one place. Maybe just accept the `.unwrap()` at that call site? ...

Reply:
It could be used in more though, and if we ever support Windows might be worth isolating?

Reply:
Then maybe just refactor then when it actually would be non-trivial?

Reply:
I prefer to see 2-3 duplicative usages before refactoring.

Reply:
(or see that such duplicative usages are actually going to happen soon)